packages/react-native-aztec/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/react-native-aztec",
-	"version": "1.52.2",
+	"version": "1.53.0",
```

Comment:
This change and a few others in the PR I believe are from the release script run for 1.53.0. It is a little odd that we include these changes, but I believe their inclusion originates from the fact that this PR branched from the `rnmobile/1.53.0` tag. I don't believe it is an issue we have to worry about, as the the changes in this PR will be squashed and release script changes will likely "disappear", as they already exist on `trunk`.